### PR TITLE
Make run.py accept .c and .wast files

### PR DESCRIPTION
**Pull request by cierniak**
See issue #19.

You can now run the script like this:

```
$ ./scripts/run.py test/hello.c --dump dd
Hello, world!

$ ./scripts/run.py dd/hello.wast
Hello, world!
```

### CODE REVIEW

File: scripts/run.py [hunk 1]
```diff
@@ -55,18 +57,22 @@ def main(args):
...
   try:
     if options.js_env:
-      s_file = os.path.join(temp_dir, basename + '.s')
-      wast_file = os.path.join(temp_dir, basename + '.wast')
+      if extension == '.c':
+        s_file = os.path.join(temp_dir, basename + '.s')
+        wast_file = os.path.join(temp_dir, basename + '.wast')
+        subprocess.check_call([WACC, '-fno-builtin', options.in_file, '-o', s_file, '-I', '.'])
+        subprocess.check_call([LINKER, s_file, '-o', wast_file])
+      elif extension == '.wast':
```
Comment (binji) [inline scripts/run.py:65]: Should add an else case. Maybe just throw?

**Comment by binji** [comment]
lgtm w/ note

File: scripts/run.py (hunk 1 above)
Comment (cierniak) [inline scripts/run.py:65]: A good idea. I added an else clause.

File: scripts/run.py [hunk 2]
```diff
@@ -55,18 +57,24 @@ def main(args):
...
+        wast_file = options.in_file
+      else:
+        raise NotImplementedError('unimplemented input file type: %s' % options.in_file)
```
Comment (binji) [inline scripts/run.py]: NotImplementedError is usually used for unimplemented functions in an abstract class. I'd just use Exception here.

Comment (cierniak) [inline scripts/run.py]: Changed!

**Pull request merged**